app/controllers/candidate_interface/unsubscribe_from_emails_controller.rb

```diff
@@ -0,0 +1,18 @@
+module CandidateInterface
+  class UnsubscribeFromEmailsController < CandidateInterfaceController
+    skip_before_action :authenticate_candidate!, :set_user_context
+
+    def unsubscribe
+      candidate = Candidate.find_by_token_for!(:unsubscribe_link, token_param)
```

Comment:
Do we handle a deleted Candidate.
i.e. We delete a candidate and they have an old email that they unsubscribe from

Reply:
If they are deleted, we would show them the 404 -- I think that is the right thing to do since they are deleted and we should behave as though we have no idea who they are.